Clarify disabling Command Log (#4655)
cypress-io/cypress-documentation: content/guides/references/troubleshooting.md

CYPRESS_NO_COMMAND_LOG=1 cypress run
```

> Disabling the Command Log can make tests faster as there is no time spent
> displaying the command log and updating it as tests run. The downside is,
> when there is a failure you cannot see what command failed.

Reviewer: ```suggestion
With this variable set, Cypress will skip rendering the Command Log entirely, and perform none of the usual DOM updates to display information about commands and statuses as the test runs. 
```

> **Note:** In headless mode screenshots and videos will not include the Command

Reviewer: > In headless mode screenshots and videos will not include the Command Log.

Really it's in any mode: open , run headed, or run headless. No command log will render when a spec is running.

```suggestion
**Note:** With this variable set, screenshots and videos will not include the Command Log
```
